src/common/TrackedOp.cc

```diff
@@ -214,7 +214,11 @@ bool OpTracker::check_ops_in_flight(std::vector<string> &warning_vector)
   //store summary message
   warning_vector.push_back("");
 
-  int slow = 0;     // total slow
+  int _slow = 0;    // total slow
+  if (!slow)
+    slow = &_slow; 
+  else
+    *slow = _slow;  // start from 0 anyway
```

Comment:
I don't see reason why 'slow' needs to be input/output parameter. I think it's better to keep it as output parameter.

The rest change looks good